Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerCommandCollectionViewCell_Tests.swift

```diff
@@ -2,25 +2,94 @@
 // Copyright © 2021 Stream.io Inc. All rights reserved.
 //
 
+import SnapshotTesting
+@testable import StreamChat
 @testable import StreamChatUI
 import XCTest
 
-class ChatMessageComposerCommandCellView_Tests: XCTestCase {
+class ChatMessageComposerCommandCollectionViewCell_Tests: XCTestCase {
+    /// Default width and height for the cell.
+    private let defaultCellSize: CGSize = CGSize(width: 250, height: 50)
+    
+    /// Default content of the view
+    private let defaultCommand = Command(name: "Default", description: "", set: "", args: "[text]")
+
     func test_defaultAppearance() {
-        let view = ChatMessageComposerCommandCellView()
-        view.command = .init(name: "Hello", description: "what", set: "is this", args: "WHAT")
-        AssertSnapshot(view, size: CGSize(width: 300, height: 40), record: true)
+        let cell = ChatMessageComposerCommandCollectionViewCell()
+        cell.commandView.content = defaultCommand
+        AssertSnapshot(cell, size: defaultCellSize)
     }
 
-    func test_defaultAppearance_withDirectMessageChannel() {}
+    func test_defaultAppearance_allAvailableCommands() {
+        let cell = ChatMessageComposerCommandCollectionViewCell()
+        cell.uiConfig.images.commandIcons.forEach { command in
+            cell.commandView.content = .init(name: command.key, description: "", set: "", args: "[text]")
+            AssertSnapshot(cell, variants: [.defaultLight], size: defaultCellSize, suffix: command.key)
+        }
+    }
 
-    func test_defaultAppearance_withNonDMChannel() {
-        // TODO: https://stream-io.atlassian.net/browse/CIS-652
+    func test_appearanceCustomization_usingUIConfig() {
+        class CustomView: ChatMessageComposerCommandCellView {
+            override func setUpAppearance() {
+                super.setUpAppearance()
+                
+                backgroundColor = .systemGray
+                commandNameSubtitleLabel.textColor = UIColor.white
+                commandNameLabel.textColor = UIColor.darkGray
+            }
+            
+            override func setUpLayout() {
+                super.setUpLayout()
+                
+                commandNameSubtitleLabel
+                    .trailingAnchor
+                    .constraint(equalTo: layoutMarginsGuide.trailingAnchor)
+                    .isActive = true
+            }
+        }
+        
+        let cell = ChatMessageComposerCommandCollectionViewCell()
+        cell.uiConfig.messageComposer.suggestionsCommandCellView = CustomView.self
+        cell.commandView.content = defaultCommand
```

Comment:
You're modifying the default `UIConfig` here. A safer way of doing this is what we do in other tests like this:
```swift
var config = UIConfig()
config.messageComposer.suggestionsCommandCellView = CustomView.self
cell.uiConfig = config
```

Reply:
✅ Resolved